Source/AppLovinAdapter.swift

```diff
@@ -27,11 +27,7 @@ final class AppLovinAdapter: PartnerAdapter {
     let partnerDisplayName = "AppLovin"
     
     /// Instance of the AppLovin SDK
-    static var sdk: ALSdk? {
-        didSet {
-            AppLovinAdapterConfiguration.sync()
-        }
-    }
+    static var sdk: ALSdk = ALSdk.shared()
```

Comment:
What about making this `let sdk: ALSdk` instead? A few `Self.sdk` spots can be simplified as just `sdk` if we do so.